packages/patternfly-4/react-charts/src/components/ChartArea/examples/ChartArea.md

```diff
@@ -85,17 +84,16 @@ import { Chart, ChartArea, ChartAxis, ChartGroup, ChartVoronoiContainer } from '
       </ChartGroup>
     </Chart>
   </div>
-</div>
+)
 ```
 
-## Cyan area chart with bottom aligned legend and axis label
```

Comment:
Why did the text change for all the titles?  Was this asked for by design?

Reply:
This text is now included in per-examples `id`s that we could write selectors for. Some titles like `Reversed bullet chart with right aligned legend -- for measures considered to be good when they are low` were unwieldy as well. The word "Simple" has been changed to "Basic" across the board.

This is really just an effort to make the titles more consistent across all components _before_ possibly writing selectors for them. @rachael-phillips Thoughts on chart example titles?

Reply:
Sounds like we need a selector ID and a separate title? We should be able to make title text as verbose as necessary to make the example clear.

Reply:
Separate ids could be done. I think overly verbose titles are bad. They also don't match the rest of the docs. I'll meet in the middle by moving the parts I remove to a description inside the example.

Reply:
Is "Simple area chart with right aligned legend" is too verbose? "Right aligned legend" doesn't feel descriptive enough, considering that is the simple example.